### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by reactivepixel**

### CODE REVIEW

File: src/routes/api/listed_apps.js [hunk 1]
```diff
@@ -1,9 +1,31 @@
...
 
   // Create
+  /**
+   * @function put
+   * @param filePath
+   * ATTN: A route or filepath with unqiue data.
+   * @param function(req,res)
+   * This is a function that takes in req and res as parameters
+   * @desc ATTN: What does this do?
+  */
```
Comment (reactivepixel) [inline src/routes/api/listed_apps.js]: Routes do not need commented in this manner as they are not actual function definitions.

File: src/routes/api/app.js [hunk 2]
```diff
@@ -18,6 +26,14 @@ module.exports = function (express) { // Setting module.exports to a new functio
 
 */
 
+/**
+ * @function get
```
Comment (jc-roll) [inline src/routes/api/app.js:33]: Desc should be under the @function not at the bottom also you need to inline the jsdoc annotation with the actual function if your confused look at my branch carsonTest or talk to me  #22

File: src/routes/api/voting.js [hunk 3]
```diff
@@ -15,87 +15,92 @@ module.exports = function(express) {
...
-  router.get('/voting/:vote_id', function(req, res) {
 /**
  * @function get
  * @description Gathers Data
- * @param req 
+ * @param req
  * Requests Data
  * @param res
- * Listens for response  
+ * Listens for response
```
Comment (jc-roll) [inline src/routes/api/voting.js:25]: This is what the QA is for I also have updated my files I am not looking forward to the re base and merging of these files if you see issues please QA next time thanks. #22

File: src/routes/api/voting.js [hunk 4]
```diff
@@ -15,87 +15,92 @@ module.exports = function(express) {
...
 
-
+  /**
+   * @function get
+   * @description This gathers data.
+   * @param req
+   * This requests data.
+   * @param res
+   * This listens for response.
+  */
```
Comment (reactivepixel) [inline src/routes/api/voting.js:67]: As we discussed the routes should not have the full function description treatment with respect to JS docs. Please continue to remove all of these on the routes.

**Comment by reactivepixel** [comment]
@kjrohr Please address _all_ the remaining comments on this pull request and proceed with the merge. This may require you to rebase master into your branch once again.

**Pull request merged**